kmedvedev/07_CSS/chess_1.html

```diff
@@ -0,0 +1,51 @@
+﻿<!DOCTYPE html>
+
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="utf-8" />
+    <title>Шахматная доска 1</title>
+    <style>
+        table {
+            background-color: black;
+            width: 600px;
+            height: 600px;
+        }
+        tr:nth-child(odd) > td:nth-child(even) {
```

Comment:
Can be one 
 tr:nth-child(odd) > td:nth-child(even),
tr:nth-child(even) > td:nth-child(odd) 
{}